Lesson_1.py

```diff
@@ -29,16 +37,25 @@ def task_one(list_of_drugs):
 print("\n")
     
 # Write a function that takes in a list of drugs, returns all unique 2-drug combinations as a list of pairs
-def task_two(list_of_combinations):
-    for idx, drug in enumerate(list_of_combinations):
-        if drug[0] == drug[1]:
-            list_of_combinations.pop(idx)
-    return list_of_combinations 
-
-unique_combinations = task_two(all_combinations)
+def make_unique_combinations(list_of_items: list) -> list:
+    """
+    Take list of items and return all unique 2-item combinations 
```

Comment:
This is not an accurate description of what you have implemented. You are describing option 1 but doing option 2. 

Option 2: `takes a list of pairs and filters out all repeats`